services/QuillLMS/spec/helpers/mobile_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe MobileHelper do
+  describe '#render_dash' do
+    it 'should return dash element if is_mobile is false' do
+      expect(helper.render_dash(false)).to eq '<div class="light-bar"></div>'
+    end
+    it 'should return nil if is_mobile is true' do
+      expect(helper.render_dash(true)).to eq nil
+    end
+  end
+  describe '#render_content' do
```

Comment:
To clean up formatting here you could use a heredoc

```ruby
let(:video_html) do
  <--HTML
    <div class='video-section'>
      <video class='lazyload' data-src='www.test.com' autoplay loop playsinline muted/>
        <p>
          Video not supported
        </p>
       /video>
    </div>
  HTML
end

expect(helper.render_content('mobile', 'mobile', 'www.test.com')).to eq video_html
```

Reply:
oh nice! much cleaner, thanks for the suggestion